_includes/reactions.njk

```diff
@@ -0,0 +1,32 @@
+<div id="reactions-list">
+  <span id="big-smile" class="reaction"><a href="/reactions/big-smile/">😀</a></span>
```

Comment:
You could just get rid of the `<a>`s, then attach the event handlers to the `<span>`s. Then, you won't need the `preventDefault` call. To get the hand icon when hovering over `.reaction`, you can add `cursor: pointer` to the CSS.

You could use the `id` to decide what to fetch.

Reply:
is that better? it seems less semantic, but maybe people don't care about that anymore

anyway, i changed it to only spans and just added a `data-href` attribute that i use to fetch

Reply:
`preventDefault()` will often eventually get you into trouble, so that's why I think it's worth it.

Re: Semanticness: It's not a link, either, so you break even there. To make it really semantic, you'd make it a `<button>`.